src/core/InputField/InputField.tsx

```diff
@@ -61,8 +61,8 @@ const InputField: React.FC<InputProps> = ({
         <InputLabel htmlFor="outlined-adornment-password">{label}</InputLabel>
         <OutlinedInput
           id="outlined-adornment-password"
-          type={showPassword ? 'text' : 'password'}
-          error={variant == 'error' ? true : false}
+          type={type === 'password' && showPassword ? 'text' : 'password'}
```

Comment:
type will just be html input types...html input automatically knows the type password

Reply:
@arkajyotiMukherjee  type={type}  is ok? do I have to write like this?

Reply:
Yes

Reply:
@arkajyotiMukherjee I tried this way but it didn't work correctly,when I write type={type} and it has been passes as 'password' clicking into icon to change type from password to text or opposite it will not work because type is already given and will not change , so maybe it will be better to write it in this way type={ showPassword? 'text' : type }
Let me know what you are thinking about this,thanks

Reply:
@AVRahul

Reply:
yes, I think in the case of the password input field we can have `type={ showPassword? 'text' : 'password' }`

Reply:
@AVRahul ok thanks, it's already was done in my last commit